dapp/src/utils/acreApi.ts

```diff
@@ -36,6 +36,7 @@ async function deleteSession() {
 
 type PointsDataResponse = {
   dropAt: number
+  isPreparingDrop: boolean
 }
```

Comment:
I looked at this [PR](https://github.com/thesis/acre-bots/pull/45) and noticed that the backend uses the `isDropReady` property. So shouldn't we use the same name?

Reply:
Addressed. Changed to `isCalculationInProgress` in both dApp and API. More context [here](https://github.com/thesis/acre-bots/pull/45#discussion_r1801259239).